src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/AmazonSqsTransport.php

```diff
@@ -90,10 +90,7 @@ public function reset()
         }
     }
 
-    /**
-     * @return MessageCountAwareInterface&ReceiverInterface
-     */
-    private function getReceiver(): ReceiverInterface
+    private function getReceiver(): MessageCountAwareInterface&ReceiverInterface
```

Comment:
Let's postpone this change to 7.0 where we can use this type for the constructor parameter and the property as well?